src/datasources/system/README.md

```diff
@@ -0,0 +1,6 @@
+# Systemlink System data source
+
+<!--
+    TODO: Write a short description of your plugin and document any extra
+    configuration that is required for development.
+-->
```

Comment:
Let's fill out the description here so we don't forget to do it later on. Doesn't need to be too involved - you can just write an overview of the plugin's features.